qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -49,3 +49,29 @@ def test_fscrypt_encrypt(self):
         # skipped for now because of not supporting features in kernel or kceph.
         self.assertEqual(proc.returncode, 0)
         self.assertIn('Passed all 26 tests', stdout)
+
+    def test_fscrypt_dummy_encryption_with_quick_group(self):
+        self.require_kernel_mount()
+
+        self.write_local_config('test_dummy_encryption')
+
+        # XXX: check_status is set to False so that we can check for command's
+        # failure on our own (since this command doesn't set right error code
+        # and error message in some cases) and print custom log messages
+        # accordingly. This will take a long time and set the timeout to 5 hours.
+        proc = self.mount_a.client_remote.run(args=['sudo', './check',
+            '-g', 'quick', '-E', './ceph.exclude'], cwd=self.xfstests_repo_path,
+            stdout=StringIO(), stderr=StringIO(), timeout=18000, check_status=False,
+            omit_sudo=False, label='running tests for dummy_encryption from xfstests-dev')
+
+        if proc.returncode != 0:
+            log.info('Command failed.')
+        log.info(f'Command return value: {proc.returncode}')
+        stdout, stderr = proc.stdout.getvalue(), proc.stderr.getvalue()
+        log.info(f'Command stdout -\n{stdout}')
+        log.info(f'Command stderr -\n{stderr}')
+
+        # Currently many test cases will be skipped dues to not supporting
+        # features, but still as successfuly.
```

Comment:
I'm not able to understand this line, do you mean to say that for the skipped tests, we will mark them successful for the time being?

Reply:
Please see https://tracker.ceph.com/issues/58126#note-9. If some futures are unsupported by kceph the test will just be skipped.

Reply:
Okay, I would like to suggest:

```
Currently, many test cases will be skipped due to unsupported features, but still will be marked as successful.
```

Reply:
Looks better and fixed it. Thanks @dparmar18